recipes-nodes/rbuilder-status-watchdog/rbuilder-status-watchdog.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          rbuilder-status-watchdog
+# Required-Start:    $network
+# Required-Stop:
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: rbuilder status watchdog
+# Description:       Checks the status of rbuilder in BuilderHub and toggles the service
+### END INIT INFO
+
+set -e
+
+SYSTEM_API_FIFO="/var/volatile/system-api.fifo"
+RBUILDER_PERSISTENT_DIR="/persistent/rbuilder"
+RBUILDER_STATUS_FLAG_FILE="${RBUILDER_PERSISTENT_DIR}/rbuilder.enabled"
+BHUB_URL="http://localhost:7937/api/l1-builder/v1/configuration"
```

Comment:
should we define this in the build env vars?

for devnet we need to change bhub URL, and it'd be nice to change that only once 🤔

Reply:
Renamed this it `BHUB_URL_PROXIED` and left a comment to make it clear it is not the external address but the one proxied locally through cvm-proxy. This URL is not supposed to be externally configured.